ygong/mosaic/submit.py

```diff
@@ -122,7 +122,7 @@ def get_experiment_run_url(experiment_name: str, run_name: str):
                                                    filter_string=f'tags.run_name = "{run_name}"',
                                                    output_format='list')
       if len(runs) == 0:
-            raise ValueError(f"run {run_name} does not exist in experiment {experiment_name}")
+            return None
```

Comment:
why returning this?

Reply:
Because we want to refresh the display(including experiment url) even before the mlflow run is ready. For example, queued status should be displayed before experiment is ready and status is RUNNING.